tracee-rules/signatures/golang/kubernetes_api_connection_test.go

```diff
@@ -38,14 +45,52 @@ func TestK8sApiConnection(t *testing.T) {
 							ArgMeta: tracee.ArgMeta{
 								Name: "remote_addr",
 							},
-							Value: map[string]string{"sa_family": "AF_INET", "sin_port": "80", "sin_addr": "1.1.1.1"},
+							Value: map[string]string{
+								"sa_family": "AF_INET",
+								"sin_port":  "80",
+								"sin_addr":  "1.1.1.1",
+							},
+						},
+					},
+				},
+			},
+			Findings: map[string]types.Finding{
```

Comment:
That's the main thing: Instead of asserting on true or false we assert on the whole Finding object, which includes event and data. Otherwise we're not catching regressions in the signature logic.